hazelcast/src/test/java/com/hazelcast/config/XmlConfigSchemaLocationTest.java

```diff
@@ -157,13 +153,11 @@ private boolean shouldSkipValidation(String nameSpaceUrl) {
     }
 
     private int getResponseCode(String url) throws Exception {
-        HttpGet httpGet = new HttpGet(url);
-        CloseableHttpResponse response = null;
-        try {
-            response = httpClient.execute(httpGet);
-            return response.getStatusLine().getStatusCode();
-        } finally {
-            IOUtil.closeResource(response);
-        }
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(URI.create(url))
+                .build();
+
+        return httpClient.send(request, HttpResponse.BodyHandlers.ofString())
```

Comment:
```suggestion
        return httpClient.send(request, HttpResponse.BodyHandlers.discarding())
```

If we aren't using the response, we shouldn't process it.